src/devtools/client/webconsole/utils/autocomplete.ts

```diff
@@ -0,0 +1,249 @@
+import * as babelParser from "@babel/parser";
+import * as t from "@babel/types";
+const { filter } = require("fuzzaldrin-plus");
+
+type ObjectPreviewProperty = {
+  value: ValueFront;
+  writable: boolean;
+  configurable: boolean;
+  enumerable: boolean;
+  name: string;
+  get?: any;
+  set?: any;
+};
```

Comment:
What do you think about putting newlines around these just to give them some separation?